Hogajama/configuration/local_configuration/03_set_rules_storage.cli

```diff
@@ -0,0 +1 @@
+/system-property=hogarama.rules.storage:add(value="${env.HOGARAMA_RULES_STORAGE:mongo}")
```

Comment:
shouldn't 'postgres' be the default value for this property? because in the hogarama_local.env we configure postgres as configuration storage and would therefore have a configuration drift between local development and production

Reply:
The reason is to stay compatible to the previous version. Principle to separate deployment from release. Just deploying the new version shouldn't change anything. You activate the new Feature by setting HOGARAMA_RULES_STORAGE in the environment.